extensions/bigtable/install_hbase_bigtable.sh

```diff
@@ -78,3 +80,53 @@ echo -e "YARN_OPTS=\"\${YARN_OPTS} -Dyarn.app.mapreduce.am.command-opts=\"${BIGT
 # the hbase classpath.
 echo -e "HBASE_OPTS=\"\${HBASE_OPTS} ${BIGTABLE_BOOT_OPTS}\"" >> "${HBASE_CONF_DIR}/hbase-env.sh"
 
+# Configure Spark
+if [ -d "/home/hadoop/spark-install/" ]; then
+    SPARK_HOME="/home/hadoop/spark-install"
+    export MASTER=spark:\/\/${MASTER_HOSTNAME}:7077
+    export SPARK_LOCAL_IP=${MASTER_HOSTNAME}
+    export SPARK_MASTER_HOST=${MASTER_HOSTNAME}
+    export SPARK_CONF_DIR=${SPARK_HOME}\/conf
+    export SPARK_MASTER_OPTS=" -Xbootclasspath/p:${ALPN_CLASSPATH}"
+    export SPARK_DAEMON_JAVA_OPTS=" -Xbootclasspath/p:${ALPN_CLASSPATH}"
+    export SPARK_EXECUTOR_OPTS=" -Xbootclasspath/p:${ALPN_CLASSPATH}"
+    export SPARK_WORKER_OPTS=" -Xbootclasspath/p:${ALPN_CLASSPATH}"
+
+    PATH=${SPARK_HOME}/bin:$PATH
+    export BIGTABLE_CLASSPATH
+    # Merge spark-install/conf/core-site.xml with its template
+    bdconfig merge_configurations \
+	--configuration_file ${HADOOP_CONF_DIR}/core-site.xml \
+	--source_configuration_file spark-core-template.xml \
+	--resolve_environment_variables    
+    
+    # Setup classpath and bootstrap classpath
+    echo -e "spark.executor.extraJavaOptions ${ALPN_JAVA_OPTS}" >> "${SPARK_HOME}/conf/spark-defaults.conf"
+    echo -e "spark.executor.extraClassPath ${GCS_CLASSPATH},${BIGTABLE_CLASSPATH},${HBASE_SERVER_CLASSPATH},${HBASE_CLIENT_CLASSPATH},${HBASE_COMMON_CLASSPATH},${HBASE_ANNOTATIONS_CLASSPATH},${HADOOP_COMMON_CLASSPATH},${GUAVA_CLASSPATH}" >> "${SPARK_HOME}/conf/spark-defaults.conf"
+    echo -e "spark.driver.extraJavaOptions ${ALPN_JAVA_OPTS}" >> "${SPARK_HOME}/conf/spark-defaults.conf"
+    echo -e "spark.driver.extraClassPath ${GCS_CLASSPATH},${BIGTABLE_CLASSPATH},${HBASE_SERVER_CLASSPATH},${HBASE_CLIENT_CLASSPATH},${HBASE_COMMON_CLASSPATH},${HBASE_ANNOTATIONS_CLASSPATH},${HADOOP_COMMON_CLASSPATH},${GUAVA_CLASSPATH}" >> "${SPARK_HOME}/conf/spark-defaults.conf"
+    echo -e "spark.jars ${GCS_CLASSPATH},${BIGTABLE_CLASSPATH},${HBASE_SERVER_CLASSPATH},${HBASE_CLIENT_CLASSPATH},${HBASE_COMMON_CLASSPATH},${HBASE_ANNOTATIONS_CLASSPATH},${HADOOP_COMMON_CLASSPATH},${GUAVA_CLASSPATH}" >> "${SPARK_HOME}/conf/spark-defaults.conf"
+
+    # Add PREFIX to env so that applications can use it to create a Spark Context
+    echo -e "export PREFIX=${PREFIX}" >> "${SPARK_HOME}/conf/spark-env.sh"
+    
+    # Configure Spark: install sbt to build Spark project
+    echo "deb http://dl.bintray.com/sbt/debian /" | sudo tee -a /etc/apt/sources.list.d/sbt.list
+    sudo apt-get update
+    sudo apt-get install -y --force-yes sbt
```

Comment:
Why do we need to install sbt here?  I think that sbt installation ought to be something we can document as a potential user action rather than something that goes into bdutil